scripts/zones/Davoi/mobs/One-eyed_Gwajboj.lua

```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+-- Area: Davoi
+-- NPC: One-Eyed Gwajboj
+-- Involved in Quest: Under Oath : PLD AF3
+-----------------------------------
+
+require("scripts/globals/settings");
+
```

Comment:
When you have a mob script that does literately nothing, it doesn't need to exist.